kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -936,7 +936,8 @@ For each `CALL*` operation, we make a corresponding call to `#call` and a state-
     rule #hasValidInitCode(INITCODELEN, SCHED) => notBool Ghasmaxinitcodesize << SCHED >> orBool INITCODELEN <=Int maxInitCodeSize < SCHED >
 
     syntax KItem ::= "#codeDeposit" MessageResult
-    rule <k> #codeDeposit(MessageResult(... gas: GAVAIL, status: STATUS)) => #refund GAVAIL ~> 0 ~> #push ...</k>
+    rule <k> #codeDeposit(MessageResult(... gas: GAVAIL, status: STATUS, data:OUT)) => #refund GAVAIL ~> 0 ~> #push ...</k>
```

Comment:
This is not quite correct. If you check against the original evm semantics, you will see it has 3 cases: success, revert, and the owise case. What you have done handles revert correctly but introduces a subtle issue if the code executed by CREATE termitates abnormally due to a reason other than the REVERT opcode.

Reply:
Makes sense. I've just pushed the case where we get an exception